sessions/TypeScript-100/Module-102-common-types/common-types.md

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
By convention, the repo doesn't use any prefix or suffix for each session folder.
Following that, seems the directory should be renamed into `102`.

The filename should also be turned into `README.md`.